### PR TITLE
python2-configparser - 3.5.0 - new package

### PR DESCRIPTION
**Pull request by JPeterMugaas**

### CODE REVIEW

File: mingw-w64-python2-configparser/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Maintainer: J. Peter Mugaas <jpmugaas@suddenlink.net>
+
+_realname=configparser
+pkgbase=mingw-w64-python-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-python2-${_realname}"
+pkgver=3.5.0
+pkgrel=1
+pkgdesc="This library brings the updated configparser from Python 3.5 to Python 2.6-3.5."
```
Comment (Alexpux) [inline mingw-w64-python2-configparser/PKGBUILD]: Fix description

File: mingw-w64-python2-configparser/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+arch=('any')
+url="https://bitbucket.org/ambv/configparser"
+license=('MIT')
+depends=('python2')
+makedepends=('python2-setuptools')
```
Comment (Alexpux) [inline mingw-w64-python2-configparser/PKGBUILD]: Wrong dependencies.

File: mingw-w64-python2-configparser/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+source=("https://pypi.io/packages/source/c/configparser/configparser-${pkgver}.tar.gz" license)
```
Comment (Alexpux) [inline mingw-w64-python2-configparser/PKGBUILD]: one entry per line

**Comment by JPeterMugaas** [comment]
OOPS!!! Where am I?

File: mingw-w64-python2-configparser/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Maintainer: J. Peter Mugaas <jpmugaas@suddenlink.net>
+
+_realname=configparser
+pkgbase=mingw-w64-python-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-python2-${_realname}"
+pkgver=3.5.0
+pkgrel=1
+pkgdesc="This library brings the updated configparser from Python 3.5 to Python 2.6-3.5. (mingw-w64)"
```
Comment (Alexpux) [inline mingw-w64-python2-configparser/PKGBUILD]: I dont understand part ````rom Python 3.5 to Python 2.6-3.5````

File: mingw-w64-python2-configparser/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+arch=('any')
+url="https://bitbucket.org/ambv/configparser"
+license=('MIT')
+depends=('python2')
```
Comment (Alexpux) [inline mingw-w64-python2-configparser/PKGBUILD]: Wrong dependency

**Comment by Alexpux** [comment]
@JPeterMugaas what about my comments?

**Comment by JPeterMugaas** [comment]
Alright, I think the package description might be correct. It is "This library brings the updated configparser from Python 3.5 to Python 2.6-3.5."  The python2 package is a backport to Python2 and later versions that didn't have the configparser modual.

**Comment by Alexpux** [comment]
@JPeterMugaas you read not all my comments seems

**Comment by JPeterMugaas** [comment]
Alright, I went ahead and changed that problematic dependency to "${MINGW_PACKAGE_PREFIX}-python2".  Sorry, I missed that comment if it what you referred to.

**Pull request merged**